examples/counter/common/src/test/kotlin/org/reduxkotlin/examples/counter/ReducerTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.reduxkotlin.examples.counter
+
+import ch.tutteli.atrium.api.cc.en_GB.toBe
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import ch.tutteli.atrium.verbs.expect
+
+
+object CounterSpek : Spek({
+
+    describe("reducers") {
```

Comment:
Why is this describe nested? Seems like there should only be one, not two

Reply:
hmm, following the js example again here.  I believe its nested here so you could have multiple reducers tested here, and nested under the reducers group.